src/includes/blogpost.njk

```diff
@@ -1,4 +1,9 @@
 <div class="blogpost">
+  <div class="blogpost-breadcrumb">
```

Comment:
This could be `nav` with an `aria-label` attr to distinguish it from other navigation.

Reply:
yeah, right. that's much better.